### ISSUE TITLE
'redirected you too many times' when accessing r studio from control panel

### ISSUE THREAD

**Issue by RobinL** [ministryofjustice/analytics-platform#84]
Occasionally users report issues like this:

> I'm having trouble opening rstudio from home. I can open the cpanel but when i try to open rstudio it says `<username>-rstudio.tools.alpha.mojanalytics.xyz redirected you too many times` 

I have also experienced this problem in the past - I've never managed to figure out a reliable way of reproducing it.

## Workaround
Usually if I close the web browser and re-open, it works again.

**Comment by davidread** [comment]
I captured it in firefox with dev tools open: ![screen shot 2018-11-14 at 15 46 29](https://user-images.githubusercontent.com/307612/48493826-90c38700-e824-11e8-970a-cf49a3525d77.png)

The URL sequence is:
```
/
/login
https://dev-analytics-moj.eu.auth0.com/authorize?prompt=none&response_type=code&client_id=<redacted>&redirect_uri=https%3A%2F%2Fdavidread-jupyter-lab.tools.dev.mojanalytics.xyz%2Fcallback&scope=openid%20openid%20email%20profile%20offline_access
/callback?error=login_required&error_description=Multifactor%20authentication%20required
/login
...
```
Auth0 should show the page where it offers GitHub login, then the MFA screen, then callback with a code. Instead it calls the app back with an error about MFA.

I found this error mentioned in the [Silent Auth docs](https://auth0.com/docs/api-auth/tutorials/silent-authentication): "login_required: The user was not logged in at Auth0, so silent authentication is not possible". I'm not clear what is the problem.

However the docs then say that when getting this error, control panel should redirect to Auth0 **but without the `prompt=none` parameter**. I tried this manually and logged in ok, so we should do this.
